index.js

```diff
@@ -1,3 +1,17 @@
-// javascript
+// Capture user's input on form submission
+let form = document.querySelector("form");
 
-alert("Hello from index.js!");
+form.addEventListener("submit", function (event) {
+  event.preventDefault();
+  console.log("Form submitted");
+});
+// Get the user's email address - userEmail (string)
+// Get the user's level - userLevel (string)
+// Get the user's hours of study userHours (number)
+// Validate the user's input
+// Check if the user has selected a level - check level entered; check selected "Beginner" or "Advanced"; conditional statements
+// Check if the user has provided an email address - check email entered, valid format (e.g., name@domain.com).
+// Check if the user has specified at least one hour of study - must be => 1
+// Check if the number of hours requested is within the allowed range - <= to mx number allowed in level
+// Calculate the total cost
+// Display the total cost to the user
```

Comment:
JavaScript: Your comments indicate a clear plan for form validation and processing. Note the techniques we use when implementing the validation logic to ensure the form data is correctly handled and provide meaningful feedback to users. These will come in handy for your own project.